gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
Her står det 8.4, men i build.gradle står det 8.3?

Jeg ser det mangler noen filer som pleier å bli oppdatert når man oppdaterer gradle wrapper. Kjørte du ./gradlew wrapper to ganger på rad? Av en eller annen grunn må man gjøre det for at alt skal oppdatere seg 🤷‍♀️

Reply:
ah, ja, jeg vurderte å gå ned til 8.3, men glemte å endre. 8.4 ser ut til å fungere, så jeg kan endre til det.
Nei, har ikke kjørt ./gradlew wrapper i det hele tatt, skal prøve nå.